package.json

```diff
@@ -786,7 +786,7 @@
                 "appService.defaultWebAppToDeploy": {
                     "scope": "resource",
                     "type": "string",
-                    "description": "The default web app to use when deploying.  Every subsequent deployment will deploy to this web app or slot."
+                    "description": "The default web app to use when deploying.  Every subsequent deployment of this project will deploy to this web app or slot.  The web app is represented with its full Azure id string.  Can be disabled by setting to \"None\""
```

Comment:
Can we combine `The default web app to use when deploying.` and `The web app is represented with its full Azure id string.`

Something like `The full Azure id of the default web app to use when deploying.`

Reply:
nit: 'workspace' instead of 'project'